docs/source/concept_models/vivarium_ciff_sam/concept_model.rst

```diff
@@ -0,0 +1,248 @@
+.. role:: underline
+    :class: underline
+
+
+..
+  Section title decorators for this document:
+
+  ==============
+  Document Title
+  ==============
+
+  Section Level 1 (#.0)
+  +++++++++++++++++++++
+  
+  Section Level 2 (#.#)
+  ---------------------
+
+  Section Level 3 (#.#.#)
+  ~~~~~~~~~~~~~~~~~~~~~~~
+
+  Section Level 4
+  ^^^^^^^^^^^^^^^
+
+  Section Level 5
+  '''''''''''''''
+
+  The depth of each section level is determined by the order in which each
+  decorator is encountered below. If you need an even deeper section level, just
+  choose a new decorator symbol from the list here:
+  https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#sections
+  And then add it to the list of decorators above.
+
+
+.. _2019_concept_model_vivarium_ciff_sam:
+
+===========================
+Vivarium ACUTE MALNUTRITION
```

Comment:
I don't think we've been using all-caps for other models, let's stay consistent.